nextroute/factory/format.go

```diff
@@ -23,6 +23,19 @@ func Format(
 	return format(ctx, options, progressioner, solutions...)
 }
 
+// FormatExt formats a solution in a basic format using the [schema.Output] to
+// format a solution.
+func FormatExt(
```

Comment:
What does `Ext` mean? Do we need to keep the other `Format` function as well. Can't we just adapt the existing `Format` function?